config/initializers/devise.rb

```diff
@@ -298,4 +343,9 @@
   # When using OmniAuth, Devise cannot automatically set OmniAuth path,
   # so you need to do it manually. For the users scope, it would be:
   # config.omniauth_path_prefix = '/my_engine/users/auth'
+
+  # authentification with pam possible
+  #config.pam_authentication = true
```

Comment:
This and the one below should probably become ENV variables so people don't have to edit source code to configure it.